src/components/side_panel/conditional_formatting/cf_editor/cf_editor.ts

```diff
@@ -570,14 +605,18 @@ export class ConditionalFormattingEditor extends Component<Props, SpreadsheetChi
     const isInvalid = valueIndex === 0 ? this.isValue1Invalid : this.isValue2Invalid;
     return {
       onConfirm: (str: string) => {
-        this.state.rules.cellIs.values[valueIndex] = str;
+        this.state.rules.cellIs.values[valueIndex] = adaptFormulaToSheet(
+          this.env.model.getters,
+          str,
+          this.props.sheetId
+        );
```

Comment:
what does this do ? When testing it never changes anything ? And I don't get why we could need to modify the ranges of the formula ?

Reply:
If you start editing while one the other sheet, the composer use the current sheet as the 'defaultRangeSheetId' (Meaning A1 = s2!A1) which should be prefixed when saving the string